src/components/explorer/IndividualVariants.js

```diff
@@ -0,0 +1,78 @@
+import React from "react";
+
+import {Table} from "antd";
+import "antd/es/table/style/css";
+
+import {EM_DASH} from "../../constants";
+import {renderOntologyTerm} from "./ontologies";
+import {individualPropTypesShape} from "../../propTypes";
+
+import { PropTypes }from "prop-types";
+// TODO: Only show variants from the relevant dataset, if specified;
+//  highlight those found in search results, if specified
+
+// const VARIANTS_COLUMNS = [
+//     {
+//         title: "ID",
+//         key: "id",
+//         render: (_, individual) => individual.id,
+//         sorter: (a, b) => a.id.localeCompare(b.id),
+//         defaultSortOrder: "ascend",
+//     },
+//     {
+//         title: "Sampled Tissue",
+//         key: "sampled_tissue",
+//         render: (_, individual) => renderOntologyTerm(individual.sampled_tissue),
+//     },
+//     {
+//         title: "Procedure",
+//         key: "procedure",
+//         render: (_, individual) => <>
+//             <strong>Code:</strong> {renderOntologyTerm(individual.procedure.code)}<br />
+//             {individual.procedure.body_site
+//                 ? <><br /><strong>Body Site:</strong> {renderOntologyTerm(individual.procedure.body_site)}</>
+//                 : null}
+//         </>,
+//     },
+//     {
+//         title: "Ind. Age at Collection",
+//         key: "individual_age_at_collection",
+//         render: (_, individual) => {
+//             const age = individual.individual_age_at_collection;
+//             return age
+//                 ? (age.hasOwnProperty("age") ? age.age : `Between ${age.start.age} and ${age.end.age}`)
+//                 : EM_DASH;
+//         },
+//     },
+// ];
+
+const IndividualVariants = ({individual}) =>
+{
+    const biosamples = (individual || {}).phenopackets.flatMap(p => p.biosamples);
+    const variantsFlat = biosamples.map(b=>b.variants).flatMap(v => v.map(_v => _v.hgvsAllele));
```

Comment:
The `hgvsAllele` is hardcoded here but it can be one of four options because it uses the dynamic schema here.
For now, it's ok but in the future, it should be changed otherwise it won't be able to display other types.

Reply:
instead of hardcoding you should get the value of {"allele_type"}